bin/inx-chronicle/src/api/stardust/history/extractors.rs

```diff
@@ -1,149 +1,194 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use std::str::FromStr;
+use std::{fmt::Display, str::FromStr};
 
 use async_trait::async_trait;
 use axum::extract::{FromRequest, Query};
-use chronicle::{db::collections::SortOrder, types::stardust::block::OutputId};
+use chronicle::{
+    db::collections::SortOrder,
+    types::{stardust::block::OutputId, tangle::MilestoneIndex},
+};
 use serde::Deserialize;
 
 use crate::api::{error::ParseError, ApiError};
 
 const DEFAULT_PAGE_SIZE: usize = 100;
+const DEFAULT_SORT_ORDER: SortOrder = SortOrder::Newest;
```

Comment:
Consider deriving `Default` for `SortOrder` (new rustc feature). You need to derive `Default` as usual and slap `#[default]` on the `Newest` variant.

Reply:
I did consider that, but other endpoints might want to have a different sorting order. I expect the `SortOrder` to be used in multiple places in the future.